driver/config/config.go

```diff
@@ -1370,6 +1371,10 @@ func (p *Config) WebAuthnForPasswordless(ctx context.Context) bool {
 	return p.GetProvider(ctx).BoolF(ViperKeyWebAuthnPasswordless, false)
 }
 
+func (p *Config) WebAuthnEnableDiscoverableCredentials(ctx context.Context) bool {
+	return p.GetProvider(ctx).BoolF(ViperKeyWebAuthnEnableDiscoverableCredentials, false)
```

Comment:
This should default to true, no? Is there a downside to enabling this?

Reply:
It's changing the UI nodes in the login flow. So I think this should be enabled by the user explicitly. We might want to enable it for new projects in Ory Cloud though.

Reply:
yes should be default-on in the new experience